### PR TITLE
Element scene properties update for CompoundParameterTypes : Fix #350

### LINKED ISSUE #350: [BUG] 3DViewer Element properties changes are not applied to the scene object
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-WEB
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

When changing an element property the scene object don't react to it
The property validation is blocking
### Logs

  - [ ] There are no logs regarding the issue
  - [ ] I have attached a log file or included the relevant section in this issue  
  
### System Configuration

- COMET version:
  - [ ] COMET-WEB: 
  - [ ] Other: 
- Web Services:
  - [ ] COMET Web Services version:

### PR DESCRIPTION
**Pull request by nabilatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-WEB-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-WEB [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-WEB-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)
### Description

Fix #350 

The CompoundParameterType changes are not blocking the element scene update

### CODE REVIEW

File: COMETwebapp/ViewModels/Components/Viewer/PropertiesPanel/PropertiesComponentViewModel.cs [hunk 1]
```diff
@@ -205,11 +208,26 @@ public async Task ParameterValueSetChanged(IValueSet valueSet)
...
             if (valueSet is ParameterValueSetBase parameterValueSetBase)
             {
+                var validationMessageBuilder = new StringBuilder();
                 var newValueArray = new ValueArray<string>(valueSet.ActualValue);
 
-                var validationMessage = ParameterValueValidator.Validate(valueSet.ActualValue.First(), this.SelectedParameter.ParameterType, this.SelectedParameter?.Scale);
+                if(this.SelectedParameter.ParameterType is CompoundParameterType compoundParameterType)
+                {
+                    var components = compoundParameterType.Component.ToList();
+                    for(var componentIndex = 0; componentIndex < components.Count; componentIndex++)
```
Comment (antoineatrhea) [inline COMETwebapp/ViewModels/Components/Viewer/PropertiesPanel/PropertiesComponentViewModel.cs:217]: Add a blank line

File: COMETwebapp.Tests/Components/Viewer/PropertiesPanel/PropertiesComponentTestFixture.cs [hunk 2]
```diff
@@ -98,5 +108,51 @@ public void VerifyThatComponentCanBeHidden()
...
+            var compoundValues = new List<string> { "1" };
+
+            var parameterValueSet = new ParameterValueSet()
+            {
+                Iid = Guid.NewGuid(),
+                ValueSwitch = ParameterSwitchKind.MANUAL,
+                Manual = new ValueArray<string>(compoundValues),
+            };
+
+            Assert.DoesNotThrowAsync(() => this.viewModel.ParameterValueSetChanged(parameterValueSet));
```
Comment (antoineatrhea) [inline COMETwebapp.Tests/Components/Viewer/PropertiesPanel/PropertiesComponentTestFixture.cs:154]: Please set does Assert inside an Assert.Multiple. Moreover, use the "Assert.That()"

**Review by antoineatrhea — CHANGES_REQUESTED** [review]

**Comment by antoineatrhea** [comment]
If possible, try to reach at least 70% coverage

**Review by jaimeatrhea — APPROVED** [review]
Approved when @antoineatrhea changes are fixed

**Review by antoineatrhea — APPROVED** [review]

**Pull request merged**